src/main/java/ev3dev/actuators/lego/motors/BaseRegulatedMotor.java

```diff
@@ -165,18 +165,18 @@ public void backward(){
      * and the position of the motors will not be maintained.
      */
     @Override
-    public void flt(boolean b) {
-        this.flt();
+    public void flt(boolean immediateReturn) {
+        doStop(COAST, immediateReturn);
     }
 
     @Override
     public void flt() {
-        this.setStringAttribute(STOP_COMMAND, COAST);
+        flt(true);

```

Comment:
This is also consistent with leJOS:
https://sourceforge.net/p/lejos/ev3/code/ci/master/tree/ev3classes/src/lejos/hardware/motor/BaseRegulatedMotor.java#l161
It even seems that `flt()` was asynchronous by default even previously.

Reply:
Nope, there is a compatibility break - the true is negated in leJOS into false. This means that leJOS was waiting for a stop by default.

Reply:
This is consistent with leJOS:
https://sourceforge.net/p/lejos/ev3/code/ci/master/tree/ev3classes/src/lejos/hardware/motor/BaseRegulatedMotor.java#l183
```
float speed = 0,
int acceleration = <acceleration>,
int limit = <no limit>,
boolean hold = FALSE,
boolean waitComplete = TRUE
```

Reply:
There is the change mentioned in the ADR - waitComplete is false by default now.